maintainers/maintainer-list.nix

```diff
@@ -12506,6 +12506,15 @@
     githubId = 1024891;
     name = "Jens Nolte";
   };
+  quentin = {
+    email = "quentin@mit.edu";
+    github = "quentinmit";
+    githubId = 115761;
+    name = "Quentin Smith";
+    keys = [{
+      fingerprint = "1C71 A066 5400 AACD 142E  B1A0 04EE 05A8 FCEF B697";
+    }];
+  };
```

Comment:
Needs go do into its own commit. `maintainers: add quentin`.

Reply:
Does it also need a separate PR, or can I put two commits in this PR?

Reply:
Fine like this.